sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/ConfigurationAsyncClient.java

```diff
@@ -391,7 +392,9 @@
      *
      * {@codesnippet com.azure.data.appconfiguration.configurationasyncclient.getSetting#ConfigurationSetting}
      *
-     * @param setting The setting to retrieve based on its key and optional label combination.
+     * @param key The key of the setting to retrieve based on its key and optional label combination.
```

Comment:
Given that we are splaying out the parameters we should change the wording here.

`The key of the setting to retrieve`
`Optional. A label used to create the key-label combination to identify the setting`.
`Optional. A date time used to retrieve the setting state at the passed time.`